myutils.py

```diff
@@ -21,25 +22,68 @@ def __init__(self, name):
         handler.setFormatter(formatter)
         self.logger = logging.getLogger(__name__)
         self.logger.addHandler(handler)
-        self.smtpObj = smtplib.SMTP('relay.suse.de', 25)
+        try:
```

Comment:
Why not just catch this exception inside of `handle_error()`?

Reply:
because in this case I will full stack trace , which in this specific case does not make much sense,   something bad happened in forked process so I don't care what is the call chain to fork a process , I just need exit code of forked process

Reply:
ah sorry I actually not correctly understand from github short view what kind of code we discussing here :)
please ignore my explanation above it is related to different try/catch . this one - I want to explicitly mute any issues with setting up send via mail - because I don't want execution to fail in case there are any issue with that